.travis.yml

```diff
@@ -3,6 +3,6 @@ os:
   - osx
 osx_image: xcode8.3
 node_js:
-  - '8'
   - '6'
-  - '4'
+  - '8'
+  - '10'
```

Comment:
Why did you change sort order? If you don't have special meaning, please fix them along with before.